File: pclassic/2016s/GalacticCivilWar/GalacticCivilWar.java
import java.util.*;
import java.io.*;
class GalacticCivilWar {
  public static void main(String[] args) throws FileNotFoundException {
    Scanner sc = new Scanner(new FileReader("GalacticCivilWarIN.txt"));
    while (sc.hasNext()) {
      int N = sc.nextInt(), K = sc.nextInt();
      int a[] = new int[N];
      for (int i = 0; i < N; i++) {
        a[i] = sc.nextInt();
      }
      int ans = galactic(a, K);
      System.out.println(ans);
    }
  }

  //Fill out the body of this method
  public static int galactic(int[] a, int K) {
    int ans = 0;

    for (int i = 0; i < a.length; i++) 
      for (int j = i + 1; j < a.length; j++) 
        if (Math.abs(a[i] - a[j]) >= K)
          ans++;

    return ans;
  }
}

File: pclassic/2016s/GalacticCivilWar/GalacticCivilWar.py
import math

#Fill out the body of this method
def different_numbers(numbers, n):
    return 0


def parse_file_and_call_function():
    with open("GalacticCivilWarIN.txt", "r") as f:
        while True:
            data = f.readline().split(' ')
            numbers = f.readline().split(' ')
            n = data[1]
            x = different_numbers(numbers, n)

            print (x)

if __name__ == "__main__":
    parse_file_and_call_function()